core/arch/x86/decode.c

```diff
@@ -709,7 +709,7 @@ read_vex(byte *pc, decode_info_t *di, byte instr_byte,
 }
 
 /* Given an instr_info_t PREFIX_EXT entry, reads the next entry based on the prefixes.
- * Note that this function does not initialise the opcode field in \p di but is set in
+ * Note that this function does not initialize the opcode field in \p di but is set in
```

Comment:
I can only support countries that are part of the European union, that excludes Britain :) No, I thought it somehow stood out since it's spelled nowhere else like that. A quick grep does find it in third_party/valgrind.